sql_helper/__init__.py

```diff
@@ -1,381 +1,40 @@
 import re
 import settings_helper as sh
-from sqlalchemy import create_engine, text, inspect
-from sqlalchemy.exc import ResourceClosedError
-from sqlalchemy.sql import sqltypes
-from os.path import isfile
-
+from sqlalchemy import create_engine
+from postgresql import PostgresqlEngine
+from mysql import MysqlEngine
+from sql import SQLEngine
 
 get_setting = sh.settings_getter(__name__)
 sql_url = get_setting('sql_url')
 connect_timeout = get_setting('connect_timeout', 5)
 rx_mysql = re.compile(r'mysql://([\S]+)')
 
+def SQL(url, connect_timeout=connect_timeout, **connect_args):
+    """An instance that can execute SQL statements on a SQL db (mysql/postgresql)
+
+    - url: connection url to a SQL db
+        - postgresql://someuser:somepassword@somehost[:someport]/somedatabase
+        - mysql://someuser:somepassword@somehost[:someport]/somedatabase
+
+    Other kwargs passed in will be passed to sqlalchemy.create_engine as
+    connect_args
+    """
+    connect_args['connect_timeout'] = int(connect_timeout)
```

Comment:
The cast to int should be unnecessary. Also here's a good spot for the sqlite fix

```
        if url.startswith('sqlite://'):
            connect_args['timeout'] = connect_timeout
        else:
            connect_args['connect_timeout'] = connect_timeout
```